src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import Loading from '../../../_common/components/loading.jsx';
+import { SeparatorLine } from '../../../_common/components/separator_line.jsx';
+import { FormRow, SubmitButton } from '../../../_common/components/forms.jsx';
+
+const AccountClosure = () => (
+    <React.Fragment>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-10 invisible'>
+            <h1>{it.L('Account closure confirmed')}</h1>
+            <p className='notice-msg'>{it.L('Accounts closed successfully. A confirmation email will be sent to your email.[_1]This page will redirect to the [_2] homepage after 10 seconds.', '<br />', it.website_name)}</p>
+        </div>
+        <div id='closure-container'>
+            <div id='main-header' className='gr-padding-30'>
+                <h1 id='heading'>{it.L('Account Closure')}</h1>
+                <p>{it.L('Closing your [_1] accounts involves closing all open positions in your accounts, and withdrawing your funds, and deactivating your accounts with [_1]', it.website_name)}</p>
```

Comment:
do we need an `and` after the first comma? since there are three items IMO and should only come after the second comma, please double check this with copywriters

Reply:
Yeah, this seems more reasonable but the design shows it with the extra `and`. I'll ask them again